backend/tests/test_tokenlist.py

```diff
@@ -0,0 +1,162 @@
+from json import loads
+from json.decoder import JSONDecodeError
+
+from httpx import AsyncClient, HTTPStatusError, Request, Response
+from pytest import MonkeyPatch, mark, raises
+
+from backend.app.libs.tokenlists import (
+    get_token_list,
+    maybe_populate_whitelist,
+    store_and_get_whitelists,
+)
+
+
+class MockResponse:
+    status_code = None
+
+    def raise_for_status(self):
+        return None
+
+    @staticmethod
+    def json():
+        return {
+            "name": "MockGecko",
+            "logoURI": "https://www.mockgecko.com/mock/path/thumb.jpg",
+            "keywords": ["mockfi"],
+            "timestamp": "2022-06-19T11:08:16.098+00:00",
+            "tokens": [
+                {"chainId": 1, "address": "0x6d6f636b5f31"},
+                {"chainId": 1, "address": "0x6d6f636b5f32"},
+            ],
+        }
+
+
+class TestTokenList:
```

Comment:
Nitpicking: I'd generally avoid classes. If you want to split code, splitting in modules is generally better.